pkg/imgpkg/imageset/tar_image_set.go

```diff
@@ -48,40 +46,32 @@ func (i TarImageSet) Export(bundleUnprocessedImageRef *UnprocessedImageRef, foun
 	i.logger.WriteStr("writing layers...\n")
 
 	opts := imagetar.TarWriterOpts{Concurrency: i.concurrency}
-	if bundleUnprocessedImageRef != nil {
-		err := ids.SetLabel(map[string]interface{}{rootBundleLabelKey: ""}, bundleUnprocessedImageRef.DigestRef)
-		if err != nil {
-			return nil, err
-		}
-	}
 
 	return ids, imagetar.NewTarWriter(ids, outputFileOpener, opts, i.logger, imageLayerWriterCheck).Write()
 }
 
-func (i *TarImageSet) Import(path string,
-	importRepo regname.Repository, registry ImagesReaderWriter) (*ProcessedImage, *ProcessedImages, error) {
+func (i *TarImageSet) Import(path string, importRepo regname.Repository, registry ImagesReaderWriter) (*ProcessedImages, error) {
 
 	reader := imagetar.NewTarReader(path)
 	imgOrIndexes, err := reader.Read()
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	}
 
-	rootBundleImageDesc, err := reader.FindByLabelKey(rootBundleLabelKey)
+	rootBundleImageDesc, err := reader.FindByLabelKey(RootBundleLabelKey)
```

Comment:
TarImageSet should not know about RootBundleLabelKey.

Reply:
ok good call. moved that logic out of tar_image_set.